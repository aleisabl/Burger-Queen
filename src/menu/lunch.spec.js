import React from 'react';
import { shallow } from 'enzyme';
import ComponentLunch from './lunch.js';
describe('ComponentLunch', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<ComponentLunch debug />);
  
    expect(component).toMatchSnapshot();
  });
});