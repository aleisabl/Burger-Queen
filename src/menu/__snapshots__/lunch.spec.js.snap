// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentLunch should render correctly in "debug" mode 1`] = `
<div
  className="lunch-div"
>
  <p>
    Loading...
  </p>
</div>
`;
